server/package.json

```diff
@@ -25,5 +26,13 @@
   "homepage": "https://github.com/Anirudh94/boreas#readme",
   "dependencies": {
     "express": "^4.14.0"
+  },
+  "devDependencies": {
+    "babel-cli": "^6.18.0",
+    "babel-preset-babili": "0.0.9",
+    "babel-preset-es2015": "^6.18.0",
+    "babelify": "^7.3.0",
+    "babili": "0.0.9",
+    "browserify": "^13.1.1"
   }
```

Comment:
Babel is a transpiler, which allows us to write in recent ECMAScript and have it compiled with older ECMAScript versions as the target for compatibility with a wider range of browsers. Babili is a code minifier plugin for Babel. Browserify allows us to use modules and import statements within JS, rather than having to use <script> tags in the HTML, for easier and better modularity in JS components, and Babelify is a Babel integration plugin for Browserify.